### PR TITLE
fix(wasm): Pivot content when using bindings

### LINKED ISSUE #5144: Issues regarding Pivot control
## Current behavior

When using a pivot with binding, after the page loads, all of the contents of all pages are stacked onto each other.
image

Problem number 2:
When using pivot controls, something similar to #5125 happens.

## Expected behavior

The expected behavior is that it normally loads the data from the binding and shows  it properly, just like UWP.

## How to reproduce it (as minimally and precisely as possible)
See this code:
http://playground.platform.uno/#aed77628

## Workaround

Workaround for binding issue: initialize a 10-100 millisecond delay from the load of the page/ViewModel to binding.
Workaround for display issue: Create a custom pivot template and get rid of the scroll viewer inside it.

## Environment

- [x] Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia
- [ ] Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia
- [ ] Uno.SourceGenerationTasks
- [ ] Uno.UI.RemoteControl / Uno.WinUI.RemoteControl
- [ ] Other: 

Nuget Package Version(s):

Affected platform(s):

- [ ] iOS
- [x] Android
- [x ] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] macOS
- [ ] Skia
  - [ ] WPF
  - [ ] GTK (Linux)
  - [ ] Tizen
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

IDE:

- [x] Visual Studio 2019 (version: Latest)

### PR DESCRIPTION
**Pull request by ajpinedam**
GitHub Issue (If applicable): #
closes #5144 

## PR Type

What kind of change does this PR introduce?
- Bugfix

## What is the current behavior?
Content of different PivotITems are being squashed together when using BIndings

## What is the new behavior?
Content is  rendered correctly

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Review by carldebilly — APPROVED** [review]

**Comment by davidjohnoliver** [comment]
related issue?

**Comment by ajpinedam** [comment]
> related issue?

@davidjohnoliver  Just added it

**Comment by ajpinedam** [comment]
| Before    |   After    |
| :--------: |  :--------: | 
| pivot_issue_before  |  pivot_issue_after  |

**Comment by Mohsens22** [comment]
Why isn't this merged?

**Comment by jeromelaban** [comment]
It isn't merged because there are missing runtime tests

**Review by jeromelaban — APPROVED** [review]

File: src/Uno.UI/UI/Xaml/Controls/Pivot/Pivot.cs [hunk 1]
```diff
@@ -57,6 +57,13 @@ protected override void OnApplyTemplate()
 			_isTemplateApplied = true;
 
 			UpdateProperties();
+
+#if __WASM__ || __SKIA__
+			//TODO: Workaround for https://github.com/unoplatform/uno/issues/5144
```
Comment (dr1rrb) [inline src/Uno.UI/UI/Xaml/Controls/Pivot/Pivot.cs:62]: Apprentely there is common comment prefix `// Uno workaround:` for this kind of stuff.

**Review by dr1rrb — APPROVED** [review]

**Pull request merged**